test/cypress/integration/17-conversations.test.js

```diff
@@ -70,7 +70,12 @@ describe('Conversations', () => {
       );
       cy.getByDataCy('conversation-preview').should(
         'contain.text',
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. De hominibus dici non necesse est. Immo alio genere; Si longus, lev...',
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. De hominibus dici non necesse est. Immo alio genere; Si longus, lev',
```

Comment:
Are we sure that it's going to be cut at `lev` and not before?

Reply:
Yes, My thinking is this cutoff happens due to the space in front of `Lorem` sometiems being there an sometimes not. So the end string can be `lev` or `levi`.

Reply:
Ok, all good then!